### PR TITLE
WIP: Integrate Showcase 2.0 in GitHub Standard Library

### PR DESCRIPTION
**Pull request by nvladimirovi**
To start the emulator:
cd add-on/multi-instance-skeleton/emulator
npm run start

### CODE REVIEW

**Comment by stoychevl** [comment]
I would suggest we rename the "emulator" folder to ".ui-emulator" - the dot notation shows that the folder will not be part of the ISO and the ui- prefix shows that the emulator emulates the UI.

File: add-on/multi-instance-skeleton/ui/package.json [hunk 1]
```diff
@@ -1,76 +1,38 @@
...
-  "author": "VMware Cloud Directory Extensions CLI",
-  "private": true,
-  "engines": {
-    "node": ">=10.14.2 <13.0.0"
+  "scripts": {
+    "clean": "rimraf dist",
+    "lint": "ng lint",
+    "build": "ng build",
+    "serve": "ng serve",
+    "deploy": "vcd-ext deploy --only=uiPlugin"
```
Comment (stoychevl) [inline add-on/multi-instance-skeleton/ui/package.json:10]: We shouldn't use anything from the old CLI should we?

**Comment by nvladimirovi** [comment]
> I also don't see the Docker file being update - we should change the Node container to v14.20

fixed.

File: add-on/multi-instance-skeleton/emulator/config.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "pluginServeList": [
+        "ui"
+    ],
+    "vcdHost": "https://vcd.cell.com",
+    "jwt": "eyJhbGciOiJSUzI1NiJ9...",
+    "scope": "provider",
```
Comment (stoychevl) [inline add-on/multi-instance-skeleton/emulator/config.json:7]: provider -> service-provider

**Review by tsimchev — APPROVED** [review]

**Review by stoychevl — APPROVED** [review]

**Pull request merged**